build/package/Dockerfile.java-toolset

```diff
@@ -0,0 +1,41 @@
+FROM registry.access.redhat.com/ubi8
+
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+# Add adoptopenjdk repo
+COPY build/package/yum.repos.d/adoptopenjdk.repo /etc/yum.repos.d/adoptopenjdk.repo
+
+# Install Adoptopenjdk
+RUN yum install -y wget && \
+    INSTALL_PKGS="adoptopenjdk-16-hotspot" && \
+    yum install -y --setopt=tsflags=nodocs $INSTALL_PKGS && \
+    yum clean all -y && \
+    exactVersion=$(ls -lah /usr/lib/jvm | grep "adoptopenjdk-16-hotspot" | awk '{print $NF}' | head -1) && \
+    alternatives --set java /usr/lib/jvm/${exactVersion}/bin/java && \
+    alternatives --set javac /usr/lib/jvm/${exactVersion}/bin/javac
+
+RUN java -version && javac -version
+
+RUN cd /opt && \
+    wget https://www-eu.apache.org/dist/maven/maven-3/3.8.1/binaries/apache-maven-3.8.1-bin.tar.gz && \
+    tar xzf apache-maven-3.8.1-bin.tar.gz && \
+    ln -s apache-maven-3.8.1 maven && \
+    ln -s /opt/maven/bin/mvn /usr/local/bin/mvn && \
+    mkdir -p $HOME/.m2 && \
+    mvn -version
+
+RUN yum -y install unzip && \
+    cd /opt && \
+    wget https://services.gradle.org/distributions/gradle-7.1.1-bin.zip -P /tmp && \
+    unzip -d /opt/gradle /tmp/gradle-7.1.1-bin.zip && \
+    ln -s /opt/gradle/gradle-7.1.1/bin/gradle /usr/local/bin/gradle && \
+    gradle -v
+
+# Container support is now integrated in Java 11, the +UseCGroupMemoryLimitForHeap option has been pruned
```

Comment:
Is this copied from the Jenkins agent? I don't think it is needed here.

Reply:
not required, I'll remove it